common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -23,7 +23,7 @@
 import io.opentracing.Tracer;
 import io.opentracing.propagation.Format;
 
-public abstract class AbstractTracer implements Tracer, Closeable {
+public abstract class AbstractTracer implements Tracer {
```

Comment:
I don't really like these changes, as they are totally unrelated to this. I will revert after I merge this PR.